src/cmd/x.ts

```diff
@@ -0,0 +1,50 @@
+import { Argv } from 'yargs'
+import { $, nothrow } from 'zx'
+import { OtomiDebugger, terminal } from '../common/debug'
+import { BasicArguments } from '../common/no-deps'
+import { cleanupHandler, otomi, PrepareEnvironmentOptions } from '../common/setup'
+
+const fileName = 'x'
+let debug: OtomiDebugger
+
+/* eslint-disable no-useless-return */
+const cleanup = (argv: BasicArguments): void => {
+  if (argv['skip-cleanup']) return
+}
+/* eslint-enable no-useless-return */
+
+const setup = async (argv: BasicArguments, options?: PrepareEnvironmentOptions): Promise<void> => {
+  if (argv._[0] === fileName) cleanupHandler(() => cleanup(argv))
+  debug = terminal(fileName)
+
+  if (options) await otomi.prepareEnvironment(debug, options)
+}
+
+export const x = async (argv: BasicArguments, options?: PrepareEnvironmentOptions): Promise<void> => {
+  await setup(argv, options)
+  const commands = argv._.slice(1).join(' ')
+  const output = await nothrow($`${commands}`)
+  output.stdout
+    .trim()
+    .split('\n')
+    .filter(Boolean)
+    .map((line) => debug.log(line))
+  output.stderr
+    .trim()
+    .split('\n')
+    .filter(Boolean)
+    .map((line) => debug.error(line))
+  process.exit(output.exitCode)
```

Comment:
I can't run this command with the debugger off, could you please allow for that?

Reply:
Don't you need output for it? You can always set `--log-level=FATAL`

Reply:
I understand you reasoning that `otomi x` is always kind of a debugging situation, however, I find it a bit inconsistent with setting log levels for the rest of the cli.

Reply:
They what do you propose for output?

Reply:
Just nothing for output expect when I set the log level to what I would expect to see, to keep it consistent with the rest.

Reply:
That doesn't seem to make sense to me. As you are running a shell script, so you expect some output right?

Reply:
Of course there is output right? Let me put it a different way, for every command I don't expect any debug output except when I set the log level...

Reply:
when I run `binzx/otomi x echo "test 123"` then I expect "test 123" to be printed, regardless of verbosity level?

Because when I run `echo "test 123"` in terminal, I expect that output, regardless of any environment variable of verbosity. Or am I missing some context? If so, let's call?